nncf/model_creation.py

```diff
@@ -141,6 +142,27 @@ def create_compressed_model(model: Module, config: NNCFConfig,
 
     for builder in compression_algo_builder_list:
         compressed_model = builder.apply_to(compressed_model)
+
+    frozen_layers = []
+    for layer in get_frozen_nncf_modules(compressed_model, convert_scopes_back_from_nncf_modules=True):
+        if should_consider_scope(str(layer[0]), target_scopes, ignored_scopes):
+            frozen_layers.append(layer)
+    if len(frozen_layers) > 0:
+        problem_algos = []
+        for builder in compression_algo_builder_list:
+            if not builder.are_frozen_layers_allowed():
+                problem_algos.append(builder.algo_name())
+        if len(problem_algos) > 0:
+            raise RuntimeError(
+                f'Frozen layers are not allowed for {problem_algos}. '
+                f'Please unfreeze them or put into the Ignored Scope.\n'
+                f'Frozen Layers: {[str(pair[0]) for pair in frozen_layers]}')
+        else:
+            logger.warn('Frozen layers detected. Using ' +
+                        reduce(lambda x, y: x + " and " + y,
```

Comment:
not sure it's a good idea to insert code when you form a string in-place. this code may fail, but the error traceback would be confusing. 
I'd recommend doing some code outside of logging or raising an error (as in the code above).